src/Functions/extractElectrodePointclouds.m

```diff
@@ -1,12 +1,29 @@
-%% extractElectrodePointclouds - Preprocess CT Data for Electrode Artifacts
+function [elecsPointcloudStruct, brainMask] = extractElectrodePointclouds(niiCT, varargin)
+% extractElectrodePointclouds - Preprocess CT Data for Electrode Artifacts
+%
+% USAGE:
+%
+%   [elecsPointcloudStruct, brainMask] = extractElectrodePointclouds(niiCT, varargin)
+%
+% INPUTS: 
+%    niiCT:         NiftiMod object of a post-operative CT image
```

Comment:
@adhusch Could you please provide the suitable descriptions of the inputs and outputs, and verify if the varargin arguments are listed properly?

Reply:
medtronicXMLPlan: Path to XML file defining a Medtronic Stealth Station S7 trajectory plan
methalThreshold: CT metal threshold value